WooCommerce/Classes/Extensions/UIApplication+Woo.swift

```diff
@@ -1,12 +1,22 @@
 import Foundation
 import UIKit
 
+// MARK: - UIApplication Utils
+//
+extension UIApplication {
+
+    /// Returns the keyWindow. Accessing `UIApplication.shared.keyWindow` is deprecated from iOS 13.
+    ///
+    var currentKeyWindow: UIWindow? {
+        UIApplication.shared.windows.filter {$0.isKeyWindow}.first
```

Comment:
Sorry if this is obvious, but I don't have enough context. 

My understanding is that the reason for deprecation was that the concept of `keyWindow` does not make sense for an iOS application anymore. 

I know it might be a bit overkill now, as it seems like we do not support multiple window, but would it make sense to take the potential scenes into account, filtering the active scene by its activationState and then grabbing its first key window?

Reply:
Exactly. The deprecation message says: `Should not be used for applications that support multiple scenes as it returns a key window across all connected scenes`.


> I know it might be a bit overkill now, as it seems like we do not support multiple window, but would it make sense to take the potential scenes into account, filtering the active scene by its activationState and then grabbing its first key window?

If I understand correct, do you mean something like this?
```let keyWindow = UIApplication.shared.connectedScenes
        .filter({$0.activationState == .foregroundActive})
        .map({$0 as? UIWindowScene})
        .compactMap({$0})
        .first?.windows
        .filter({$0.isKeyWindow}).first```

Reply:
Yes? I mean, if we take the scene into account, that looks like the way to go, but I am not sure if we need to complicate things that much...

Reply:
So, I'd say to maintain the current behavior since we are sure it's still working like before. I'm not sure if in the future, if we will ship a mac OS app, we should handle this in a different way. :)